Solutions/CrowdStrike Falcon Endpoint Protection/Data Connectors/Connector_Syslog_CrowdStrikeFalconEndpointProtection.json

```diff
@@ -2,7 +2,7 @@
     "id": "CrowdStrikeFalconEndpointProtection",
```

Comment:
Please rename all static text `Azure Sentinel` to `Microsoft Sentinel` in this file i.e. Connector_Syslog_CrowdStrikeFalconEndpointProtection.json file

Reply:
@v-amolpatil, Please check now